src/aspire/utils/rotation.py

```diff
@@ -369,25 +369,65 @@ def generate_random_rotations(
     @staticmethod
     def angle_dist(r1, r2, dtype=None):
```

Comment:
Hey @garrettwrong, just tagging you here in case you want to take a look at the changes to this method that were made after you approval.